runelite-client/src/main/java/net/runelite/client/plugins/mining/MiningPlugin.java

```diff
@@ -355,12 +339,6 @@ public void onWallObjectSpawned(WallObjectSpawned event)
 			case GOLD_VEIN: // Arzinian gold vein
 			case GOLD_VEIN_5990: // Gold vein
 			case GOLD_VEIN_5991: // Gold vein
-			{
```

Comment:
why is this being removed? isnt it important?

Reply:
Oops, I didn't mean to delete that. I originally added the new object IDs here, but later realized they are wall game objects, not wall objects - and then I deleted more than my new code!